io.catenax.message_header/1.0.0/MessageHeaderAspect.ttl

```diff
@@ -0,0 +1,126 @@
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.message_header:1.0.0#> .
+
+:MessageHeaderAspect a samm:Aspect ;
+   samm:preferredName "Message Header Aspect"@en ;
+   samm:description "Aspect model describing the message header which must be included in every message"@en ;
+   samm:properties ( :header ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:header a samm:Property ;
+   samm:preferredName "header"@en ;
+   samm:description "Contains information common to all messages"@en ;
+   samm:characteristic :HeaderCharacteristic .
+
+:HeaderCharacteristic a samm:Characteristic ;
+   samm:preferredName "Header Characteristic"@en ;
+   samm:description "Characteristic describing the common shared aspect Message Header"@en ;
```

Comment:
same here, more verbose description with some details for users, who want to know when to apply the header would be helpful. You can decide, where it makes semantically more sense to place the details of the description, either in the property, entity or characteristic. I would assume the characteristic is the most appropriate place.

Reply:
Long-form description of the message header has been added to the entity